File: easy/position.clj
(require '[clojure.string :as str])
(doseq [item (for [st (str/split-lines (slurp (first *command-line-args*)))]
  (#(= (bit-test (Integer/parseInt (first %))
                 (- (Integer/parseInt (nth % 1)) 1))
       (bit-test (Integer/parseInt (first %))
                 (- (Integer/parseInt (nth % 2)) 1)))
(str/split st #",")))] (prn item))

File: easy/position.lua
for line in io.lines(arg[1]) do
  _, _, x, p1, p2 = string.find(line, "(%d+),(%d+),(%d+)")
  p1, p2 = 2^(p1-1), 2^(p2-1)
  print((x % (p1 + p1) >= p1) == (x % (p2 + p2) >= p2))
end
